docs/content.zh/getting-started/with-flink-cdc.md

```diff
@@ -0,0 +1,27 @@
+---
+weight: 6
+title: With Flink CDC
+bookHref: "https://nightlies.apache.org/flink/flink-cdc-docs-stable/docs/try-flink-cdc"
```

Comment:
This link points to an empty page

Reply:
Had an offline discussion with @GOODBOY008 and we decided to add a page on CDC side, so we keep the link as is.